api/src/main/java/org/eclipse/microprofile/lra/client/LRAClient.java

```diff
@@ -379,4 +277,22 @@ URL updateCompensator(URL recoveryUrl, Class<?> resourceClass, URI baseUri, Stri
      * @return the current LRA (can be null)
      */
     URL getCurrent();
+
+    /**
+     * Lookup the status of an LRA
+     *
+     * @param lraId the LRA whose status is being requested
+     *
+     * @return the status of the the LRA.
+     *
+     * @throws NotFoundException if the LRA no longer exists
+     *
+     * @throws UnknowableException if the status is unknowable
```

Comment:
I miss here a case where LRA client doesn't know current state right now but can potentially provide this information in the future. Some use-cases -- coordination is down and then comes up again, routing slip is received by this service by the next call (deducing from javadoc of UnknowableException)